src/components/card/card.ts

```diff
@@ -1,5 +1,5 @@
 import {Component, View, ViewEncapsulation} from 'angular2/core';
-import {CONST_EXPR, Type} from 'angular2/src/facade/lang';
+import {CONST_EXPR} from "angular2/src/facade/lang";
```

Comment:
Use single-quotes? 
(you can configure WebStorm to use single instead of double-quotes whenever it inserts something in `Editor > Code Style > TypeScript > Other > Generated Code`